build.ps1

```diff
@@ -83,5 +82,4 @@ if ($env:SKIP_TESTS -ceq "False") {
       `-`-experimental_profile_include_target_label `
       `-`-build_event_json_file test-events.json `
       `-`-build_event_publish_all_actions `
-      `-`-experimental_execution_log_file ${ARTIFACT_DIRS}/logs/test_execution_windows_$env:SYSTEM_JOBATTEMPT.log `
```

Comment:
The binary execution log is basically useless: as far as I know nobody ever looks at it, and I don't seem to be able to build the parser for it. Trying to get the JSON execution log crashes the build on an OOM.